tests/Tests/Cat/CatMaster/CatMasterApiTests.cs

```diff
@@ -36,6 +36,7 @@
 namespace Tests.Cat.CatMaster
 {
 	[SkipVersion(">=2.0.0", "CatMaster API was renamed to CatClusterManager in 2.0.0 release")]
+	// TODO
```

Comment:
TODO?

Reply:
Indeed. Why not create CatClasterManagerApiTests now?

Reply:
It was created in https://github.com/Bit-Quill/opensearch-net/pull/29/files#diff-e4a3470817fb4093031a3a1c2b530667b6a501d14243bf57f82f09ab07511cb5. I left TODO mark to check that nothing missed and forgot to delete it. Thank you for finding!

Reply:
Fixed in 285868649.